backend-engineer/importer/README.MD

```diff
@@ -0,0 +1,109 @@
+**Importer Challenge - Backend Engineering @ Vizzuality**
+
+---
+
+Instructions for completing and submitting the challenge:
+First things first. This coding challenge is about creating a space for you to share with us how you work and reason, what things you care about when doing coding work, and how you approach problem-solving.
+
+As such, through this challenge we are not expecting to check if you know the finest algorithms, have all the right answers to a given situation or if you are the best coder in the world. We believe there are no right or wrong answers, so please make yourself comfortable and focus on what you know best.
+
+## ✨ Overview:
+
+Design and implement a robust data ingestion and querying system using a modular backend architecture. 
+The goal is to simulate a real-world import pipeline that scales well and meets performance, quality, and maintainability standards.
+
+Please read carefully all the instructions. If you feel that something is not clear, please reach out, or feel free to make an educated guess!
+
+
+## 📆 Time estimate
+
+Please submit your ideas within 7 days of receiving the challenge, so that we have enough time to review your solution with the rest of the team
+before the next interview. 
+
+Based on our previous experience, and the available tooling nowadays, we believe that it should take no more than 8-10h.
+But do not let this to be a limiting factor to you. Good enough means good enough, but it's up to you how much do you want to polish it
+
+---
+
+## 1. Requirements
+
+You are required to implement a small system that allows importing and querying environmental data.
+
+This system must include:
+
+### A. **CSV Import Microservice**
+
+* Accepts a CSV upload (e.g., via POST endpoint).
+* Parses and validates the file.
+* Stores the data in a **relational database**.
+* Performs basic aggregation (e.g., record count, min/max for numeric fields) post-import.
+
+### B. **Query API Microservice**
+
+* Exposes a REST API to query the imported dataset.
+* Must support:
+    * Filtering by any field (query params)
+    * Pagination and sorting
+* Must return results quickly.
+
+### C. **Infrastructure & Tooling**
+
+* Use a single **public** GitHub repository (no need to create separate repositories for the services)
```

Comment:
we may want to drop the public part - understandably, people may not want to announce to the world that they are in a job application process, or may not want to create an ad-hoc anonymous/pseudonymous GH account. We routinely end up with submissions for which we discuss invitations to private repos, so I'd outright remove this bit. No need to add anything else, as P&C will take care of coordinating access in any case (or candidates send tarball, etc.)

Reply:
I do agree on this. The only single point of this was to allow the candidate to run GH action in order to perform the testing pipeline goal.

I thought there was no way no way to use gh actions on private repos freely but it seems that there is a limited number of minutes available. We should check this, but let's leave it like this for now. I have removed the public part